model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -123,16 +137,9 @@ message PTransform {
   //
   // For primitive transform in the Runner API, this is required, and the
   // payloads are well-defined messages. When the URN indicates ParDo it
-  // is a ParDoPayload, and so on.
-  //
-  // TODO: document the standardized URNs and payloads
-  // TODO: separate standardized payloads into a separate proto file
```

Comment:
we didn't actually do this second TODO, ParDoPayload et al are still here. I guess we don't want to move them anymore?

Reply:
Yeah, I closed the related BEAM-4180 since the effort isn't worth it and the `Payload` term does provide meaning that its meant to be embedded.